docs/configuration.md

```diff
@@ -2531,6 +2531,36 @@ newlines.inInterpolation
 - `avoid`: attemps to avoid breaks within the spliced code, regardless of line overflow
 - `oneline`: formats the splice on a single line, or breaks after `${` if overflows
 
+### `newlines.ignoreInSyntax`
+
+This boolean parameter controls whether to ignore newlines in strings (or other
```

Comment:
Shouldn't this only be used for strings? It's a bit unclear what `other possibly multi-line tokens` could entail. Overall, would that be a setting for that one user?

Reply:
1. the code doesn't really specify which tokens could have embedded newlines; other possible options are comment and xml.
2. as to the target audience: i would definitely use it in our codebase, not a big fan of ignored embedded newlines myself.

Reply:
@tgodzik any additional thoughts on this?

Reply:
I think I am missing the exact meaning here. `whether to ignore newlines in strings` yet the newlines are outside the strings,  so this only affects strings if they are inside some other srtucture?

Reply:
hmm... let me add a longer explanation :)